templates/.eslintignore

```diff
@@ -1,3 +1,9 @@
 .eslintrc.js
 *.config.js
-build
\ No newline at end of file
+build
+
+# Next.js app ignore paths
+.next
+next-env.d.ts
+pages/_app.js
+pages/api/hello.js
```

Comment:
These shouldn't be here - we don't want to ignore linter errors in these for the students